vmm/src/lib.rs

```diff
@@ -1000,8 +1000,8 @@ impl Vmm {
                 .map_err(CreateRateLimiter)?;
 
             let vm_fd = self.vm.get_fd();
-            cfg.take_tap()
-                .ok_or(NetDeviceNotConfigured)
+            cfg.open_tap()
```

Comment:
Why not propagate the error `open_tap` returns? It will help debug the reason that made it failed.
Also if `open_tap()` fails the error the user will see will be
"The net device configuration is missing the tap device.".
Is this accurate?